src/mediapipe_calculators/ovmscalculator.proto

```diff
@@ -0,0 +1,49 @@
+//*****************************************************************************
+// Copyright 2023 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
```

Comment:
To be removed

Reply:
Decided to leave this calculator for cross verification between using pure C-API and Adapter.